### PR TITLE
Resize to minimum columns on warning: set minimum width

### PR DESCRIPTION
**Pull request by krlakwa**

### CODE REVIEW

File: src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js [hunk 1]
```diff
@@ -95,8 +95,9 @@ class DatagridCell extends React.Component {
   }
 
   getStyle() {
-    const { width } = this.props;
-    return { width, minWidth: '2rem', maxWidth: width };
+    const { width, editable } = this.props;
+    if (editable) return { width, maxWidth: width };
+    return { width, minWidth: '4.625rem', maxWidth: width };
```
Comment (renaudtertrais) [inline src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js]: actually this should be the opposite no ?

Comment (krlakwa) [reply src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js]: then the columns will not display correctly

Comment (renaudtertrais) [reply src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js]: minWidth: '4.625rem' was only for editable columns, for other ones we should keep the  '2rem

File: src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js [hunk 2]
```diff
@@ -95,8 +95,8 @@ class DatagridCell extends React.Component {
   }
 
   getStyle() {
-    const { width } = this.props;
-    return { width, minWidth: '2rem', maxWidth: width };
+    const { width, editable } = this.props;
+    return editable ? { width, maxWidth: width } : { width, minWidth: '2rem', maxWidth: width };
```
Comment (renaudtertrais) [inline src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js]: can you move this minWidth in SCSS like you've done for editable ? then no need for ternary

File: src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.scss [hunk 3]
```diff
@@ -60,6 +62,7 @@
   margin: auto;
   padding-left: $spacing-x-small;
   padding-right: $spacing-x-small;
+  overflow: visible;
```
Comment (renaudtertrais) [inline src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.scss]: sure about that ?

File: src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js (hunk 2 above)
Comment (krlakwa) [reply src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js]: ok, i'll fix it

File: src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.scss (hunk 3 above)
Comment (krlakwa) [reply src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.scss]: nope, should be removed

**Pull request merged**